src/kafka-net/KafkaConnection.cs

```diff
@@ -90,11 +91,14 @@ public Task SendAsync(KafkaDataPayload payload, CancellationToken token)
         /// <typeparam name="T">A Kafka response object return by decode function.</typeparam>
         /// <param name="request">The IKafkaRequest to send to the kafka servers.</param>
         /// <returns></returns>
+
+
         public async Task<List<T>> SendAsync<T>(IKafkaRequest<T> request)
         {
             //assign unique correlationId
             request.CorrelationId = NextCorrelationId();
 
+            _log.DebugFormat("SendAsync CorrelationId:{0}", request.CorrelationId);
```

Comment:
yes only for debug purpose,
it help understand when the request(single or beach) get into the connection queue when it real send  and when it we get a reprocess back.

but it need to be refactor to make more seance